prow/gerrit/reporter/reporter.go

```diff
@@ -103,46 +104,48 @@ func (c *Client) ShouldReport(pj *v1.ProwJob) bool {
 }
 
 // Report will send the current prowjob status as a gerrit review
-func (c *Client) Report(pj *v1.ProwJob) error {
+func (c *Client) Report(pj *v1.ProwJob) ([]*v1.ProwJob, error) {
 	// If you are hitting here, which means the entire patchset has been finished :-)
 
 	clientGerritRevision := client.GerritRevision
 	clientGerritID := client.GerritID
 	clientGerritInstance := client.GerritInstance
 	pjTypeLabel := kube.ProwJobTypeLabel
+	gerritReportLabel := client.GerritReportLabel
 
 	// list all prowjobs in the patchset matching pj's type (pre- or post-submit)
 	selector := labels.Set{
 		clientGerritRevision: pj.ObjectMeta.Labels[clientGerritRevision],
 		pjTypeLabel:          pj.ObjectMeta.Labels[pjTypeLabel],
+		gerritReportLabel:    pj.ObjectMeta.Labels[gerritReportLabel],
```

Comment:
let's assert pj has `gerritReportLabel` label to be defensive. (There are other gerrit deployments, so we only want to add the label to selector if it exists)

Reply:
done

Reply:
(don't include the label when the label is empty :-p)

Reply:
done